docs/current_docs/features/dagger.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I'm not sure this is the best place for the module. We use screen recordings in other sections of the docs as well (eg. cloud traces), so maybe it would be better to have the main recorder module in the parent `docs` directory, with separate functions for features, traces, etc?